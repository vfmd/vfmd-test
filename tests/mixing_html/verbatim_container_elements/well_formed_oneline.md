## pre

Some text <pre>Pre content</pre>

### Outside pre

## style

Some text <style>Style content</style>

### Outside style

## script

Some text <script>Script content</script>

### Outside script

