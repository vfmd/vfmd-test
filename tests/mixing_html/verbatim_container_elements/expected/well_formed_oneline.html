<h2>pre</h2>
Some text <pre>Pre content</pre>

<h3>Outside pre</h3>
<h2>style</h2>
Some text <style>Style content</style>

<h3>Outside style</h3>
<h2>script</h2>
Some text <script>Script content</script>

<h3>Outside script</h3>
